src/utils/apiService.ts

```diff
@@ -325,6 +325,41 @@ function getProjects(): Promise<Array<Project>> {
   });
 }
 
+function getProject(projectId: string) {
```

Comment:
was there a reason why the response is pre-filled?

Reply:
Just for quick testing, will implement actual api call in the next steps.